api/specs/webserver/v0/openapi-auth.yaml

```diff
@@ -0,0 +1,172 @@
+openapi: 3.0.0
+info:
+  title: auth API
+  version: 0.1.0
+  description: authentication and authorization
+servers:
+  - description: API server
+    url: '/v0'
+paths:
+
+  /auth/register:
+    post:
+      operationId: auth_register
+      requestBody:
+        description: user registration
+        content:
+          application/json:
+            schema:
+              $ref: './components/schemas/registration.yaml#/RegistrationType'
+        required: true
+      responses:
+        '200':
+          description: User has been succesfully registered.
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/log_message.yaml#/LogMessageEnveloped'
+        '400':
+            $ref: './openapi-auth.yaml#/components/responses/DataError_BadRequest_400'
+        '409':
+            $ref: './openapi-auth.yaml#/components/responses/DataError_Conflict_409'
+        '422':
+            $ref: './openapi-auth.yaml#/components/responses/DataError_UnprocessableEntity_422'
+        default:
+          $ref: './openapi-auth.yaml#/components/responses/DefaultErrorResponse'
+
+  /auth/login:
+    post:
+      operationId: auth_login
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: './components/schemas/login.yaml#/LoginFormType'
+      responses:
+        '200':
+          description: Succesfully logged in
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/log_message.yaml#/LogMessageEnveloped'
```

Comment:
not sure why you respond with logmessages?

Reply:
The main reason is to transmit info messages to the final user from the server. For example, when the server sends a confirmation email (there are at least 3 of those for reset different parts)

This is not new in this PR.